querysql/logrusmssql.go

```diff
@@ -88,14 +88,21 @@ func LogrusMSSQLLogger(logger logrus.FieldLogger, defaultLogLevel logrus.Level)
 
 func logrusEmitLogEntry(logger logrus.FieldLogger, level logrus.Level) {
 	switch level {
-	case logrus.DebugLevel:
```

Comment:
These changes are not part of the exception bug, but they are nice to have.  The change simply extend the types of logs we can handle.